### ISSUE TITLE
Implement "reduce_spatial" process for the WCPS backend

### ISSUE THREAD

**Issue by pierocampa** [Open-EO/openeo-spring-driver#71]
As per title.

I understand that the [*reduce_spatial*](https://processes.openeo.org/#reduce_spatial) process is a shortcut for 2 (3) cascaded [*reduce_dimension*](https://processes.openeo.org/#reduce_dimension) on the 2 (3) spatial dimensions of a cube, though it is still handy to have it.

**Comment by clausmichele** [comment]
Agree. Especially since concatenating 2 reduce_dimension does not always produce the same result (e.g. when computing the standard deviation).
